### DISCUSSION TITLE
Typed property App\Livewire\MyForm::$startsAt must not be accessed before initialization

### DISCUSSION THREAD

**Discussion by ph7jack** [livewire/livewire#8473] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.12

### Laravel version

v11.7.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

If you try to set null to a typed property in a livewire form, it will throw an error about property initialization. It doesn't happen to non typed properties.

The ability to set a value from the js side is really important for creating alpine components
Thanks

### Repository or Wirebox URL

https://wirebox.app/b/4lk5d

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment, marked as answer]
Fix: https://github.com/livewire/livewire/pull/8474

**Reply by calebporzio** [reply]
Thanks @PerryvanderMeer !
